src/migrations/1744398181271_scene-stream-ttl.ts

```diff
@@ -0,0 +1,20 @@
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate'
+
+export const SceneStreamTTLColumns: ColumnDefinitions = {
+  streaming: { type: 'boolean', notNull: true, default: false },
+  start_streaming_at: { type: 'bigint', notNull: false }
```

Comment:
What do you think about naming the column as:
```suggestion
  streaming_start_time: { type: 'bigint', notNull: false }
```